src/com/projectkorra/projectkorra/firebending/FireBlast.java

```diff
@@ -202,7 +222,24 @@ public void progress() {
 				furnace.setBurnTime((short) 800);
 				furnace.setCookTime((short) 800);
 				furnace.update();
-			} else if (BlazeArc.isIgnitable(this.player, block.getRelative(BlockFace.UP))) {
+			} else if (block.getType() == Material.SMOKER && this.powerFurnace) {
+				final Smoker smoker = (Smoker) block.getState();
+				smoker.setBurnTime((short) 800);
+				smoker.setCookTime((short) 800);
+				smoker.update();
+		    } else if (block.getType() == Material.BLAST_FURNACE && this.powerFurnace) {
+		    	final BlastFurnace blastF = (BlastFurnace) block.getState();
+				blastF.setBurnTime((short) 800);
+				blastF.setCookTime((short) 800);
+				blastF.update();

```

Comment:
We could potentially have blue fire set a longer time?

Reply:
I considered this but I decided that this should be something thats configurable, and I purposefully omitted it with hopes of adding it in a later update.